### DISCUSSION TITLE
Theme extension not showing up because `presets` don't exist in schema

### DISCUSSION THREAD

**Discussion by haseebintegriti** [kinngh/shopify-node-express-mongodb-app#90] [Q&A]
After deploying the extension (s:e:deploy) it shows in the partner dashboard I turned it on from there but not showing in the theme editor.

**Comment by kinngh** [comment]
Do you have `presets` in your Liquid Schema for Theme Extension? Or are you talking about extension not showing up in Partner Dashboard?

**Reply by haseebintegriti** [reply]
Extension is showing up in the partner dashboard. 
![image](https://github.com/kinngh/shopify-node-express-mongodb-app/assets/133798856/1e079157-a99f-480a-b5db-bfdf69353c43)

But the extension is not showing up inside the theme editor to add under any section. e.g adding star-rating on product page or any other page. 
![image](https://github.com/kinngh/shopify-node-express-mongodb-app/assets/133798856/dda9e9d1-0d25-4a93-abb6-3b63820c8ec6)

**Reply by kinngh** [reply]
Like I stated, you need to add a `presets` entry in your schema. Please read the docs: https://shopify.dev/docs/themes/architecture/sections/section-schema#presets
